201-virtual-wan-with-all-gateways/create-vms-for-vwan-and-inspect-routes.ps1

```diff
@@ -0,0 +1,184 @@
+﻿#region Initialize Environment, Authenticate to Azure AD and setting Resource Parameters 
```

Comment:
remove this file, you can provide a template if you want to provision additional resources